frontend/apps/pwa/src/components/FormPicker.tsx

```diff
@@ -55,30 +94,24 @@ const FormPickerContainer: FC<FormPickerContainerProps> = ({
     : allForms;
 
   const callback = useCallback(
```

Comment:
Not part of this pr, but I've seen this a lot. `useCallback` is useful if the function is expensive to create or will cause lots of unnecessary rerenders. In most cases `useCallback` has more overhead than just creating the function on each render.

Reply:
Same goes for `useMemo` and `React.memo`